### PR TITLE
Candidate

### PR DESCRIPTION
**Pull request by Climbatize**
# Description
Merge successful release 2.7.0 into master

## Links
[MINI-2304](https://jira.rakuten-it.com/jira/browse/MINI-2304)

# Checklist
- [X] I have read the [contributing guidelines](CONTRIBUTING.md).
- [X] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [X] I wrote/updated tests for new/changed code
- [X] I removed all sensitive data before every commit, including API endpoints and keys
- [X] I ran `fastlane ci` without errors

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -1,5 +1,5 @@
 PODS:
-  - MiniApp (2.5.0):
+  - MiniApp (2.6.0):
```
Comment (ts-leojoseph-r) [inline Podfile.lock:2]: Isn't this supposed to be 2.7.0 🤔 ? We have 2.7.0 released in Pods already right?

**Review by ts-leojoseph-r — APPROVED** [review]
Looks good to me :+1: Just one comment, please check it.

**Review by corycaywood — COMMENTED** [review]
Looks good! 👍

**Review by corycaywood — APPROVED** [review]

File: Podfile.lock (hunk 1 above)
Comment (Climbatize) [reply Podfile.lock:2]: yeah, this is just podlock updated before the release, we don't have to worry about this

**Pull request merged**